torch/csrc/jit/mobile/import.cpp

```diff
@@ -509,5 +518,20 @@ mobile::Module _load_for_mobile(
   }
 }
 
+void _load_extra_only_for_mobile(
+    const std::string& filename,
+    c10::optional<at::Device> device,
+    ExtraFilesMap& extra_files) {
+  std::unique_ptr<FileAdapter> rai = std::make_unique<FileAdapter>(filename);
+  auto observer = torch::observerConfig().getModuleObserver();
+  auto instance_key = std::rand();
+  if (observer) {
+    observer->onEnterLoadModel(instance_key);
```

Comment:
nit: shall we fire the same onEnterLoadModel event if this is only to load extra meta without intent to run the inference? (would this affect the metric readings?)